android/src/app/organicmaps/MapFragment.java

```diff
@@ -101,36 +106,40 @@ public void onCreate(Bundle b)
   @Override
   public void onStart()
   {
+    Logger.d(TAG);
     super.onStart();
     mMap.onStart();
-    Logger.d(TAG, "onStart");
   }
 
+  @Override
   public void onStop()
   {
+    Logger.d(TAG);
```

Comment:
Now the function is printed automatically, right?

P.S. Do you log AA system parameters?

Reply:
Yes and yes.
In `NavigationSession.onCreateScreen()`